src/test/resources/net/masterthought/cucumber/location_formats.json

```diff
@@ -0,0 +1,113 @@
+[
+  {
+    "uri": "features/selenium.feature",
+    "id": "test",
+    "keyword": "Feature",
+    "name": "test",
+    "description": "",
+    "line": 1,
+    "elements": [
+      {
+        "id": "test;test1",
+        "keyword": "Scenario",
+        "name": "test1",
+        "description": "",
+        "line": 2,
+        "type": "scenario",
+        "steps": [
+          {
+            "keyword": "Given ",
```

Comment:
as asked before - do you need this steps for test? if not then delete and leave only those data that you are going to validate